lib/api/src/grpc/proto/telemetry_internal.proto

```diff
@@ -0,0 +1,140 @@
+syntax = "proto3";
+
+import "collections.proto";
+
+package qdrant;
+option csharp_namespace = "Qdrant.Client.Grpc";
+
+message GetPeerTelemetryRequest {
+    uint64 peer_id = 1; // The peer id to ask for telemetry
+    uint32 details_level = 2; // The level of detail needed
+}
+
+message GetPeerTelemetryResponse {
+    reserved 1; // id
+    reserved 2; // app
+    map<string, CollectionTelemetry> collections = 3; // Mapping from collection name to its telemetry
+    ClusterTelemetry cluster = 4; // Telemetry about the cluster and peers
+    reserved 5; // requests
+    reserved 6; // memory
+    reserved 7; // hardware
+}
```

Comment:
So this basically follows the same structure as our existing telemetry, but it is very limited.

Is there a strong argument for doing it this way, versus implementing a whole new kind of telemetry with it's own fields?

Reply:
Yes, and it is made to replicate it closely. 

We already have the information we need from existing telemetry, so we can reuse the collectors we already have for internal service.